eslint.config.mjs

```diff
@@ -16,6 +16,7 @@ export default tseslint.config(
   {
     rules: {
       'no-loss-of-precision': 'off',
+      'no-throw-literal': 'error',
```

Comment:
I can remove this if it's not something that you want added to eslint. But it will prevent new instances of this from being introduced.

Reply:
No, it's good